tests/url.js

```diff
@@ -69,6 +69,29 @@ describe('Compile the CSS of an HTML page passed by URL', () => {
         });
     });
 
+    it('Deals with local options.stylesheets when using URLs', function (done) {
+        const localStylesheetPath = path.join(__dirname, '/input/main.css');
+
+        this.timeout(25000);
+        uncss(['http://giakki.github.io/uncss/'], {
+            stylesheets: [path.join('file:', localStylesheetPath)]
```

Comment:
I still have my same concern mentioned above, about how we reference local files: https://en.wikipedia.org/wiki/File_URI_scheme

Namely, should be `file://host/path/`, or in the case of local files, should be `file:///path/`.